src/pages/klage/fritekstForAvvistKlage/AvvistKlage.tsx

```diff
@@ -0,0 +1,184 @@
+                );
+            });
+        });
+    };
+
+    const iGyldigStatusForÅAvvise = (k: Klage) =>
+        k.status === KlageStatus.VILKÅRSVURDERT_BEKREFTET_AVVIST || erKlageAvvist(k);
+
+    if (!iGyldigStatusForÅAvvise(props.klage)) {
+        return (
+            <div className={sharedStyles.feilTilstandContainer}>
+                <Alert variant="error">{formatMessage('feil.ikkeRiktigTilstandForÅVurdere')}</Alert>
+                <Link
+                    to={Routes.klage.createURL({
+                        sakId: props.sakId,
+                        klageId: props.klage.id,
+                        steg: KlageSteg.Formkrav,
+                    })}
+                >
+                    {formatMessage('knapp.tilbake')}
+                </Link>
+            </div>
+        );
+    }
+
+    return (
+        <ToKolonner tittel={formatMessage('avvistKlage.tittel')}>
+            {{
+                left: (
+                    <form onSubmit={handleSubmit(handleBekreftOgFortsettSubmit)}>
+                        <Label className={styles.resultatLabel}>{formatMessage('avvistKlage.resultat')}</Label>
+                        <div className={styles.fritesktOgVisBrevContainer}>
+                            <Controller
+                                control={control}
+                                name={'fritekstTilBrev'}
+                                render={({ field, fieldState }) => (
+                                    <Textarea
+                                        {...field}
+                                        label={
+                                            <div className={styles.fritekstLabelOgHjelpeTekstContainer}>
+                                                <Label>{formatMessage('avvistKlage.brevTilBruker')}</Label>
+                                                <HelpText>
+                                                    <Label className={styles.hjelpetekst}>
+                                                        {formatMessage('avvistKlage.brevTilBruker.hjelpetekst')}
+                                                    </Label>
+                                                </HelpText>
+                                            </div>
+                                        }
+                                        value={field.value ?? ''}
+                                        error={fieldState.error?.message}
+                                    />
+                                )}
+                            />
+                            <Button
+                                type="button"
+                                variant="secondary"
+                                onClick={() => {
+                                    const values = getValues();
+
+                                    onSeBrevClick(values.fritekstTilBrev ?? '');
+                                }}
```

Comment:
Her brukes (litt forkorta)
`onClick={() => onSeBrevClick(getValues().fritekstTilBrev ?? '')}`
Mens på linje 138 har vi 
`onClick={() => handleOnLagre(watch('fritekstTilBrev') ?? '')}`

Foreslår å bruke samme begge steder for å opprettholde konsistens 😎 
(og da ville jeg brukt `getValues` siden den ikke trigger rerender, og blir kalt på nytt på hver onClick uansett)

Reply:
fixed :)